app/views/cclow/geography/climate_targets/index.html.erb

```diff
@@ -1,87 +1,59 @@
-                  <a href="https://www4.unfccc.int/sites/NDCStaging/pages/Party.aspx?party=<%= @geography.iso %>">
-                    <%= @geography.name %>'s INDC
-                  </a>
-                </div>
-              </div>
-            </li>
-          <% end %>
-        </ul>
       </div>
-    </li>
 
-    <li>
-      <div class="climate-targets-section">
-        <div class="nav">
-          <h5>National laws and policies</h5>
-          <span class="counter"><%= pluralize(@climate_targets.count, 'target') %></span>
-        </div>
-        <ul>
-          <% @climate_targets.each do |target| %>
-            <li>
-              <div>
-                <%= target.description&.html_safe %>
+      <% @list.each do |sector, targets| %>
+        <a href="<%= "/cclow/geographies/#{@geography.id}/climate_targets/#{sector}" %>" class="columns item active">
+          <div class="column is-one-thirds sector-name"><%= sector %></div>
+          <div class="column is-one-thirds value">
+            <% if targets['ndc'].present? %>
+              <div class="current-circle">
+                <%= targets['ndc'] %>
               </div>
-              <div class="tags">
-                <% target_tags = [target.sector&.name, target.target_type&.humanize]
-                   target_tags << "Target year: #{target.year}" if target.year
-                   target_tags << "Base year: #{target.base_year_period}" if target.base_year_period
-                %>
-                <%= target_tags.compact.join(' | ') %>
+            <% else %>
+              <div class="empty-circle"></div>
+            <% end %>
+          </div>
+          <div class="column is-one-thirds value">
+            <% if (targets['law'] || 0) + (targets['policy'] || 0) %>
+              <div class="current-circle">
+                <%= (targets['law'] || 0) + (targets['policy'] || 0)%>
               </div>
-              <% if target.legislations.any? %>
-                <div class="source">Source:
-                  <% target.legislations.each do |legislation|%>
-                    <div class="name">
-                      <% path = legislation.policy? ? 'cclow_geography_policy_path' : 'cclow_geography_law_path' %>
-                      <%= link_to legislation.title, send(path, @geography, legislation) %>
-                    </div>
-                    <p class="meta">
-                    <span>
-                      <% info = [legislation.legislation_type&.humanize, legislation.date_passed&.year] %>
-                      <%= info.compact.join(' | ') %></span>
-                    </p>
-                  <% end %>
-                </div>
-              <% end %>
-            </li>
-          <% end %>
-        </ul>
-      </div>
-    </li>
+            <% else %>
+              <div class="empty-circle"></div>
+            <% end %>
+          </div>
+        </a>
+      <% end %>
+
+      <% (LawsSector.select(:name).map(&:name) - @list.keys).each do |sector| %>
```

Comment:
we should sort this alphabetically and maybe even just pass the keys to the query, instead of getting everything and then removing the ones not needed.